wire-runtime/src/main/java/com/squareup/wire/WireOutput.java

```diff
@@ -50,11 +50,12 @@
 // OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
 
 import java.io.IOException;
+import okio.BufferedSink;
 
 /**
  * Utilities for encoding and writing protocol message fields.
  */
-public final class WireOutput {
```

Comment:
... I guess we're lacking support for varint in `BufferedSink`. Still, I think it might be simpler to just have callsites call the static methods.